docs/swagger/spec.yml

```diff
@@ -21,64 +21,67 @@ x-top-matter:
-
+          By default, imported scenes are private to the user that uploaded them and scenes generated from public data sources are discoverable by all users.
+          Scene permissions will eventually expand to include making imported scenes discoverable to all users or just users within an organzation.
   - name: Images
     description: |
-      A single image, with 1 or more bands
+      Images are single or multi-band rasters that can be viewed and edited as a single unit but are always associated with a parent scene. They are sometimes synonymous with their parent scenes and othertimes just one of several images associated with their parent scenes. One common example of an image is a multispectral satellite image; another is a land classification raster.
     further-description:
       - title: Image permissions
         content: |
-          Images can be shared publicly, with your organzation, or privately.
-          All user uploaded Images are by default private, and Images generated from publicly available data are public.
+          Images respect the permissions applied to their parent scenes.
   - name: Uploads
-    description: User uploads
+    description: Uploads are datasets that a user or application has attempted to upload to Raster Foundry (successfully or unsuccessfully).
     further-description:
-      - title: Image permissions
+      - title: Uploads permissions
         content: |
-          All user uploads are private
+          All user uploads are private to that user.
   - name: Datasources
     description: |
-      Datasources contain information common to all images uploaded through them such as composites and default color corrections
+      "Datasources contain information common to all scenes associated with them, such as the number of bands to expect in images associated with that datasource. A datasource can be specific (e.g. a particular earth observation satellite) or generic (e.g. a 3-band sensor on a UAV). All scenes must have an associated datasource."
+
+      "Currently, datasources cannot be created through the API and must be requested (<a href="https://github.com/azavea/raster-foundry/issues" target="_blank">preferably as a Github issue in the source repository</a>)."
+
+      "In the future, users will be able to manage the creation, ammendment, and deletion of datasources directy through the API. Also, users will be able to assign transformations like color correction, band composites, or color maps to datasources as a default for themselves and/or their organizations."
   - name: Tokens
     description: |
-      Tokens are used to access the API, either through a frontend or an api client.
+      Tokens are used to securely identify users and applications accessing the API.
   - name: Tools
     description: |
-      Tools are used to apply transformations to Rasters
+      Tools are algorithms that can be applied to projects manually or programmatically.
```

Comment:
maybe automatically instead of programmatically? what does this actually mean right now? IE do we need to insert a note about future support?